templates/base.html

```diff
@@ -0,0 +1,80 @@
+<!doctype html>
+<html lang="{{config.default_language}}">
+    <head>
+        <meta charset="utf-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <!-- Begin Jekyll SEO tag v2.8.0 -->
+        <title>{{config.title}} | {{config.description}}</title>
+        <meta name="generator" content="Zola v0.17.2">
+        <meta property="og:title" content="{{config.title}}">
+        <meta property="og:locale" content="en_US">
+        <meta name="description" content="{{config.description}}">
+        <meta property="og:description" content="{{config.description}}">
+        <link rel="canonical" href="/">
+        <meta property="og:url" content="/">
+        <meta property="og:site_name" content="">
```

Comment:
This seems slightly strange - maybe it's right?

Reply:
I've created an issue to track the contents of <head> in base.html (#12)